src/context/servicesContext/servicesContext.jsx

```diff
@@ -20,7 +20,7 @@ const ServiceContext = createContext(initialDataState);
 
 const ServiceProvider = ({ children }) => {
   const { isAuthorized, authToken } = useAuth();
-  const { videos } = useData();
+  const { videoState } = useData();
   const { showToast } = useToast();
 
   const [state, dispatch] = useReducer(dataReducer, initialDataState);
```

Comment:
since there are a two contexts - data context and service context, it took me a while to co-relate serviceContext to dataReducer. I got confused why you were managing service state with data reducer